elements/bulbs-lightbox/bulbs-lightbox-examples.js

```diff
@@ -0,0 +1,27 @@
+import './bulbs-lightbox.scss';
+import './bulbs-lightbox-test.scss';
+
+let examples = {
+  element: 'bulbs-lightbox',
+  examples: {
+    'Bulbs LightBox': {
+      render: () => {
+        return `
+          <bulbs-lightbox>
```

Comment:
@jmelvnsn opted to make this a semantic component as opposed to `is="bulbs-lightbox"`